core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -103,6 +103,9 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def isEnableChunk(): Boolean =
     sqlConf.getConfString(TiConfigConst.ENABLE_CHUNK, "true").toLowerCase.toBoolean
 
+  private def isUseTiFlash(): Boolean =
+    sqlConf.getConfString(TiConfigConst.USE_TIFLASH, "false").toLowerCase.toBoolean
+
```

Comment:
delete unused same config in `class TiConfiguration`?

```
  public boolean isUseTiFlash() {
    return useTiFlash;
  }
```

Reply:
We may use it in future.